src/main/java/service/BookService.java

```diff
@@ -15,13 +17,22 @@ public void createBook(String bookname, String author, int year,int isbn) {
         bookRepo.addBook(book);
     }
 
-    public boolean findBook(String name,int isbn) {
-        Book book = bookRepo.getBook(name,isbn);
-        return book != null;
+    public ArrayList<Book> findBooks(String name, String author) {
+        Book book = bookRepo.getBook(name, author);
+        if (book != null){
+            return bookRepo.findBooks(book);
+        }
+        return null;
+    }
```

Comment:
Please revisit this method:
- Its going twice to the database to look for books, one in line 21 and again in line 23
- This method returns a null if there is no books, it should return an empty list as the consumers are expecting.
- `bookRepo.findBooks(book)` findBooks in the books repo should receive a book name and author, not a book